nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/PropertyCommentManager.java

```diff
@@ -421,33 +424,35 @@ protected String getCommentContainerPath(CoreSession session, String commentedDo
         });
     }
 
-    protected DocumentRef getAncestorRef(CoreSession session, DocumentModel documentModel) {
-        return CoreInstance.doPrivileged(session, s -> {
+    @Override
+    public DocumentRef getAncestorRef(CoreSession coreSession, DocumentModel documentModel) {
+        return CoreInstance.doPrivileged(coreSession, session -> {
             if (!documentModel.hasSchema(COMMENT_SCHEMA)) {
                 return documentModel.getRef();
             }
-            DocumentModel ancestorComment = getThreadForComment(s, documentModel);
+            DocumentModel ancestorComment = getThreadForComment(session, documentModel);
             return new IdRef((String) ancestorComment.getPropertyValue(COMMENT_PARENT_ID));
         });
     }
 
-    protected DocumentModel getThreadForComment(CoreSession session, DocumentModel comment)
+    protected DocumentModel getThreadForComment(CoreSession coreSession, DocumentModel comment)
```

Comment:
Why this change `session` -> `coreSession`? If it's to be consistent with other methods, you can rename the CoreSession parameter `s`.